cmd/ak/cmd/builds/delete.go

```diff
@@ -21,7 +21,7 @@ var deleteCmd = common.StandardCommand(&cobra.Command{
 		if err != nil {
 			return err
 		}
-		if b == nil {
+		if !b.IsValid() {
```

Comment:
could b be null? e.g. not valid and not found are different errors

Reply:
it's not found, hence why it's Invalid. It replaced nil.

Reply:
@ig-ra see the subtlety in the resolver functions documentation: functions return (`obj, id, err`) - it's possible that they return (`nil, id, nil`) which means that the ID is syntactically correct but not found, and that's why `!obj.IsValid()` means "not found"